src/lib/expansion-panel/expansion-panel.ts

```diff
@@ -109,6 +119,14 @@ export class ExpansionPanelComponent extends BaseComponent implements IExpansion
   @FoundationProperty()
   public declare useAnimations: boolean;
 
+  /** Sets whether an internal button and associated interactions are attached. */
+  @FoundationProperty()
+  public declare type: ExpansionPanelType;
+
+  /** Gets/sets the internal toggle button's label. */
+  @FoundationProperty()
+  public declare accessibleLabel: string;
```

Comment:
Do these class fields actually need the `declare` keyword?  I thought that was only for [providing a more specific typing](https://www.typescriptlang.org/docs/handbook/2/classes.html#type-only-field-declarations) on fields inherited from a base class, but let me know if there's another usage I'm not aware of.

Reply:
I think you're right. I'll remove them.